hack/bump-cnao.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2021 Red Hat, Inc.
+
+set -e
```

Comment:
can we add x?
```suggestion
set -xe
```

Reply:
I think it will add noise, but will add

Reply:
Its pretty noisy, it prints the yamls that it download (to a variable)
and so on, you sure we want it ?
can you run locally please (unless you did), see how it looks like and suggest ?

Reply:
I get you. I won't insist on it if you it brings more than than good